packages/js/src/ui/Inbox.tsx

```diff
@@ -31,7 +31,21 @@ const InternalInbox = () => {
   return (
     <div class={style('novu', 'root')}>
       <div class="nt-text-2xl nt-font-bold">{t('inbox.title')}</div>
-      <BellContainer />
+      <div class="nt-absolute nt-top-1/2 nt-left-1/2 nt-transform -nt-translate-x-1/2 -nt-translate-y-1/2">
```

Comment:
✏️ Maybe `nt-inset-0 nt-m-auto` here instead of top,left and translate ?

Reply:
Yes!